src/cogent3/app/io_new.py

```diff
@@ -45,6 +49,62 @@
 __status__ = "Alpha"
 
 
+@define_app
+def pickle_it(data: SerialisableType) -> bytes:
+    """attempts to call a to_rich_dict() method, fallback to
+    pickling entire object"""
+    with contextlib.suppress(AttributeError):
+        data = data.to_rich_dict()
+
+    return pickle.dumps(data)
+
+
+@define_app
+def unpickle_it(data: bytes) -> SerialisableType:
+    return pickle.loads(data)
+
+
+@define_app
+class compressed:
```

Comment:
I think name `compress` is better for this app.

Reply:
Because the other app name is `pickle_it` not 'pickled'.

Reply:
Done